wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,100 +1,84 @@
-
-                                             b. Go to Settings » Preferences, under Chatting » General, check "Hide join and part Messages"                                                                                                                                                                            |
-| [ircII](http://www.eterna.com.au/ircii/)  | /ignore \* crap                                                                                                                                                                                                                                                          |
-| [Irssi](http://www.irssi.org)             | /ignore -channels \#somechannel \* JOINS PARTS QUITS                                                                                                                                                                                                                     |
-| [Weechat](http://www.weechat.org)         | /filter add irc\_smart\_weechat irc.somename.\#somechannel irc\_smart\_filter \*  
-
-                                             **Note:** somename is the name you gave when you added the IRC server to Weechat.                                                                                                                                                                                         |
-| [KVIrc](http://www.kvirc.net)             | Refer to [this thread](http://www.kvirc.ru/forum/?topic=609.0) in the official KVIrc forums.                                                                                                                                                                             |
-| [mIRC](http://www.mirc.com/)              | Tools » Options » pick "IRC". Click the "Events..." button. Change the "joins", "parts", "quits", and "nicks" to your desired settings: "In Status" or "Hide" are good options [1](http://i.clintecker.com/disable-irc-msgs.html).                                       |
-| [Quassel IRC](http://www.quassel-irc.org) | Right click on the chatting window, then choose Hide Events » Join/Part/Quit.                                                                                                                                                                                            |
-| [XChat](http://www.xchat.org)             | Right-click on the tab you want to change. In the submenu of the channelname, there's a toggle-item "Show join/part messages", simply turn this off. Or you type /set irc\_conf\_mode 1 [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels. |
+If your client is not listed here refer to its documentation, most clients have a way to do this.
 
-If your client isn't listed here refer to its documentation, most clients have a way to do this.
 
 Frequently Asked Questions (FAQ)
 ================================
```

Comment:
Use heading level 2, there should only be one heading level 1, that being the title.
```
Frequently Asked Questions (FAQ)
--------------------------------
```